frontend/src/assets/string-to-tri-colour-palatte.js

```diff
@@ -0,0 +1,53 @@
+/* Based on https://github.com/brandoncorbin/string_to_color */
```

Comment:
Can you move this files to /helpers/ and add unit tests for it?

Reply:
Would adding tests for that helper involve removing the `JobCard.test.js` file? That function is currently the only thing being tested for that component currently.

Reply:
Eh, I mean if the functions are tested in the jobcard then its probably fine to not test. Still would prefer it in a /helpers/ file though.

Reply:
NVM i see that it was moved. Resolving conversation.